### ISSUE TITLE
Bugs with "paused" condition

### ISSUE THREAD

**Issue by ariel-phet** [phetsims/circuit-construction-kit-common#770]
Observed while testing https://phet-dev.colorado.edu/html/circuit-construction-kit-ac/1.0.0-dev.41/phet/circuit-construction-kit-ac_en_phet.html

While checking resonant circuits with inductors in parallel I saw the following very odd behavior (easily repeatable). This behavior also occurs with a single inductor, but two inductors in parallel make the behavior more dramatic.

Steps to reproduce:

1. Set up a resonant LC circuit as shown in the video (a single inductor in the circuit will work)
2. Charge the capacitor (using switch connected to the battery)
3. Isolate the capacitor (open the switch connected to the battery)
4. Get the resonant circuit going (close the switch for the LC circuit)
5. Pause the sim
6. Bring an inductor into the play area
7.  The resonant circuit will begin oscillating even though the sim is paused and the chart will begin moving up and down but not moving forward in time

https://user-images.githubusercontent.com/4051174/139594584-ff90a403-3728-4662-ae2b-ba0270d48002.mp4

**Comment by samreid** [comment]
Good discovery @ariel-phet.  I believe this is caused by our decision in https://github.com/phetsims/circuit-construction-kit-common/issues/572 to simulate updates while paused by running the model with a tiny dt:

```js
  // Run a paused clock at this rate, see https://github.com/phetsims/circuit-construction-kit-common/issues/572
  // Still need physics to update, like capacitors clearing and electrons updating.  But don't propagate time very far!
  // dt * 60 times/sec * 60 sec/minute * 60 minutes/hour  * 24 hours/day * 365 days /year
  // dt = acceptable level = 1E-3 in one year = 0.001
  // dt = 1E-3/60/60/60/24/365 = 5.284965330627432e-13
  PAUSED_DT: 5E-13,
```

However, we limit how low dt can go in the model TimestepSubdivisions, so these ideas are at odds--it's still running the model at larger dt.  We previously investigated other mechanisms for updating the circuit physics when the sim is paused, see https://github.com/phetsims/circuit-construction-kit-common/issues/572#issuecomment-705940000 and the following comments.  But it seems like we need to reopen that investigation.  I'll start on that but @arouinfar and @ariel-phet please interrupt me if you have a better recommendation.

**Comment by samreid** [comment]
I tested the "UPDATE 2" approach and it seemed OK in testing, so I published a dev version with that idea.  @ariel-phet can you please review in https://phet-dev.colorado.edu/html/circuit-construction-kit-ac/1.0.0-dev.42/phet/circuit-construction-kit-ac_en_phet.html

**Comment by ariel-phet** [comment]
@samreid I am still seeing some issues.

Building a resonant circuit similar to the one above

1. Set up a resonant LC circuit as shown in the video
2. Charge the capacitor (using switch connected to the battery)
3. Isolate the capacitor (open the switch connected to the battery)
4. Get the resonant circuit going (close the switch for the LC circuit)
5. Pause the sim
6. Open the LC circuit switch, close it, then open it again
7. "Play" the sim
8. Close the LC circuit switch, for me this action freezes the sim irrevocably

https://user-images.githubusercontent.com/4051174/140386848-92b2d2df-3739-47d6-8bcf-c7f3b3a66cbf.mp4

**Comment by samreid** [comment]
We'll move the above problem to a new issue, it seems this issue can be closed.

**Comment by samreid** [comment]
New issue opened, closing.

**Comment by samreid** [comment]
We just saw a capacitor charge while the sim was paused.

**Comment by samreid** [comment]
During the design meeting, we all observed a capacitor charge while the sim was paused.  However, I'm having difficulty reproducing the problem (testing in master).  I tested a number of simple circuits with capacitors as well as cases like https://github.com/phetsims/circuit-construction-kit-common/issues/770#issuecomment-961247642.  I'll ask for help reproducing this bug in the next dev version.

**Comment by samreid** [comment]
@arouinfar or @ariel-phet can you figure out a way to get the sim to run or a capacitor to charge while paused in version?  https://phet-dev.colorado.edu/html/circuit-construction-kit-ac/1.0.0-dev.46/phet/circuit-construction-kit-ac_en_phet.html

**Comment by KatieWoe** [comment]
I'm seeing this fairly easily in dev.48
![pausedcharge](https://user-images.githubusercontent.com/41024075/141012073-712315e9-e809-4def-b3ae-c4c11d09dbd7.gif)

**Comment by Nancy-Salpepi** [comment]
For dev.48, I am also able to charge the capacitor while the sim is paused using the setup in https://github.com/phetsims/circuit-construction-kit-common/issues/770#issuecomment-961247642
![chargedcapacitor](https://user-images.githubusercontent.com/87318828/141013616-b06c81e4-b484-45ce-a6dc-0eca3d971667.gif)

**Comment by ariel-phet** [comment]
@samreid see QA results above

**Comment by samreid** [comment]
The sim is supposed to update while paused. For instance, consider this light bulb circuit:

![Kapture 2021-11-12 at 13 50 28](https://user-images.githubusercontent.com/679486/141532874-ade94218-174f-475c-9cc1-3c44f5a7583c.gif)

If the circuit was closed and the light bulb did not light, the sim would be in an inconsistent state and seem buggy.  Therefore we added a rule that would do a discrete update even when paused.  The buggy behavior referred to in https://github.com/phetsims/circuit-construction-kit-common/issues/770#issuecomment-961469232 was a continuous capacitor charging over several seconds, not a discrete immediate update.  @KatieWoe or @Nancy-Salpepi have you seen the continuous/animated charging over time while paused?  If not, I think this issue can be closed.
